schema/default/UserDefine.fbs

```diff
@@ -14,9 +14,22 @@ enum BorderMode : byte {
     CLAMP,
     REFLECTION
 }
+enum PoolMode : byte {
+    MaxPool=0,
+    AvePool
+}
 
 table GridSample {
     mode:SampleMode;
     paddingMode:BorderMode;
     alignCorners:bool=false;
 }
+
+table RoiAlign {
+    pooledWidth:int;
+    pooledHeight:int;
```

Comment:
RoiAlign Parameter Can be modified from RoiPooling. Just add other parameter to table RoiPooling

Reply:
I know they have some parameters in common, but it's strange to use `RoiPoolingT` directly in ROIAlign op.It's hard to understand at first look. or will be better using the RoiPooling table as a parameter in RoiAlign like follow?
```
table RoiAlign {
    roiPoolingParam:RoiPooling;
    samplingRatio:int=-1;
    aligned:bool=false;
    poolMode:PoolMode=AvePool;
}
```

Reply:
You can rename RoiPooling to RoiParameters

Reply:
1